services/index.html

```diff
@@ -3,101 +3,157 @@
+            <p>{% t services-software_creation-embedded_craftsmen.subtitle %}</p>
+            
+            <h4>{% t services-software_creation-self_contained_teams.title %}</h4>
+            <p>{% t services-software_creation-self_contained_teams.subtitle %}</p>		
+
+            <h4>{% t services-software_creation-continous_delivery.title %}</h4>
+            <p>{% t services-software_creation-continous_delivery.subtitle %}</p>
+
+            <h4>{% t services-software_creation-ux_ui_design.title %}</h4>
+            <p>{% t services-software_creation-ux_ui_design.subtitle %}</p>
+        </div>
+
+      </div>
+
     </div>
 
-    <div class="col-md-6">
-      <h3 class="subject-header margin-bottom-40">{% t services-expert_advice-software_architecture_and_design.title %}</h3>
-      {% t services-expert_advice-software_architecture_and_design.subtitle %}
+    <!-- Expert advice section-->
+    <div class="col-md-4 " id="expert-advice">
+      <div class="panel panel-red equal-height-column">
+
+        <div class="panel-heading" style="background-image:url(/assets/img/custom/services/service-bg-2.png);">
+          <h3 class="panel-title">{% t services-expert_advice.title %}</h3>
+        </div>
+
+        <div class="panel-body">
+            <h4>{% t services-expert_advice-enterprise_architecture.title %}</h4>
+            <p>{% t services-expert_advice-enterprise_architecture.subtitle %}</p>
+
+
+            <h4>{% t services-expert_advice-software_architecture_and_design.title %}</h4>
+            <p>{% t services-expert_advice-software_architecture_and_design.subtitle %}</p>
+
+            <h4>{% t services-expert_advice-continous_delivery.title %}</h4>
+            <p>{% t services-expert_advice-continous_delivery.subtitle %}</p>
+
+            <h4>{% t services-expert_advice-xp.title %}</h4>
+            <p>{% t services-expert_advice-xp.subtitle %}</p>
+        </div>
+      </div>
+
     </div>
 
+    <!-- training section -->
+    <div class="col-md-4" id="training">
+      <div class="panel panel-red equal-height-column">
+
+      <div class="panel-heading" style="background-image:url(/assets/img/custom/services/service-bg-2.png);">
+        <h3 class="panel-title">{% t services-training_coaching.title %}</h3>
+      </div>
+
+      <div class="panel-body">
+          <h4>Software Development Practices</h4>
+          <p>We offer taining to help develpers adopt practices such as Test Driven Development, Continuous Integration, and Release Automation. We can also customise our trainings according to the needs of your team. <a href="#">See our courses ...</a></p>
+
+          <h4>Software Architecture and Cloud</h4>
+          <p>We will review your current soution and provide training on archicture and technologies specific to your project. For example, we can train your team to evolve your system towards a Microservices based approach utilising the most suitable offerings from AWS. </p>
```

Comment:
We will review your current solution and provide training on architecture and technologies specific to your project. For example, we can train your team to evolve your system towards a Microservices based approach utilising the most suitable offerings from AWS.